commands/build.go

```diff
@@ -279,6 +362,7 @@ func runBuild(ctx context.Context, dockerCli command.Cli, options buildOptions)
 		return err
 	}
 
+	done := timeBuildCommand(mp, options.Attributes(dockerCli))
```

Comment:
Make this `(build)metricsAttributes(b, dockerCli, options) attribute.Set`.

Don't need `options.newBuilder()` or other methods on `options`.

Inside the function, read the `b.Driver` directly from input without the struct/once. If you want then for the hash you can leave the lazy evaluation, but for just reading the driver property, there is no need for this overhead.

The call could also be inside `timeBuildCommand`.

Reply:
Made the change. This isn't called inside of `timeBuildCommand` as there's another section of code that's going to use these same attributes in another PR.